Cloud Application Manager/Analytics/CloudApplicationManagerAnalyticsUI.md

```diff
@@ -1,41 +1,108 @@
 {{{
   "title": "CenturyLink Cloud Optimization Introduction",
-  "date": "06-01-2017",
-  "author": "Chris Meyer",
+  "date": "12-13-18",
+  "author": "Ben Swoboda",
   "attachments": [],
   "contentIsHTML": false
 }}}
 
-### Introduction
-The Cloud Application Manager [Analytics UI](https://analytics.cam.ctl.io) is a location where users can view cost savings recommendations, best practices evaluations and cross capability environment configuration reporting for cloud providers that are being monitored as part of our [Cloud Optimization](https://www.ctl.io/cloud-application-manager/cloud-optimization) offering.   
+### Overview
+In Cloud Application Manager's [Analytics Site](https://analytics.cam.ctl.io) users can view cost savings recommendations, best practices evaluations and cross capability environment configuration reporting. Additionally, users gain direct access to **Advanced Analytics**.
 
+This document describes how to gain access to Analytics and provides a breakdown of the different components that are available: Savings, Best Practices, Security, Cost, and Advanced Analytics.
 
-### Overview
-This document describes how to access the Analytics UI and provides a breakdown of the different views that are available in each of the main components of the UI which consists of Savings and Best Practices.
+### Prerequisites
+One of the following use cases must have occurred successfully:
+* The provider is enabled for [Managed Services](https://www.ctl.io/knowledge-base/cloud-application-manager/managed-services/#1)
```

Comment:
This needs to be a relative link to the .md file

Reply:
Makes sense to me.